### PR TITLE
RUN-1061 - Fix Sub Menu Position and Adjust Spacing, Font Size

### PR DESCRIPTION
**Pull request by mishingo**
**Is this a bugfix, or an enhancement? Please describe.**
Bugfix: In reference to https://github.com/rundeckpro/rundeckpro/issues/2657 the submenu on the main menu was misaligned.

**Describe the solution you've implemented**
Addressing the size of the menu helps the visibility of the menu on smaller sized screens, without the need to have scrolling on the menu items, which can be difficult to see. 

image

### CODE REVIEW

**Comment by mishingo** [comment]
Before: 
image

**Comment by mishingo** [comment]
After:
image

File: rundeckapp/grails-app/views/menu/_sysConfigNavMenu.gsp [hunk 1]
```diff
@@ -16,10 +16,10 @@
...
 <%@ page import="org.rundeck.core.auth.AuthConstants" %>
 
-<g:set var="authAclRead" value="${auth.resourceAllowedTest(
+<g:set var="authRead" value="${auth.resourceAllowedTest(
         type: AuthConstants.TYPE_RESOURCE,
-        kind: AuthConstants.TYPE_SYSTEM_ACL,
-        action: [AuthConstants.ACTION_READ, AuthConstants.ACTION_ADMIN, AuthConstants.ACTION_APP_ADMIN, AuthConstants.ACTION_OPS_ADMIN],
+        kind: AuthConstants.TYPE_SYSTEM,
+        action: [AuthConstants.ACTION_READ, AuthConstants.ACTION_ADMIN, AuthConstants.ACTION_APP_ADMIN],
         any: true,
```
Comment (gschueler) [inline rundeckapp/grails-app/views/menu/_sysConfigNavMenu.gsp:23]: this seems to be reverting an unrelated change

**Review by carlosrfranco — APPROVED** [review]
LGTM
